src/coreclr/jit/lower.cpp

```diff
@@ -5308,11 +5308,15 @@ bool Lowering::LowerUnsignedDivOrMod(GenTreeOp* divMod)
             }
             else if (type != TYP_I_IMPL)
             {
+#ifdef TARGET_ARM
```

Comment:
@pentp do you by any chance remember why this special case for ARM32 exists here?